res/css/views/location/_LocationPicker.scss

```diff
@@ -37,3 +37,8 @@ limitations under the License.
         max-height: 140px;
     }
 }
+
+.mx_LocationPicker_error {
+    color: red;
```

Comment:
this should be a themed colour

Reply:
all the other red errors are hardcoded currently, and meanwhile there are a tonne of hardcoded colours flying around:

```
matthew@shadowfax css % egrep -r 'color: (#|[^\$])' | wc -l
     126
```

...so given it's an edge error case, i'm inclined to leave it be and have de-hardcoding colours as a separate PR for someone in future.